aplib.net-demo/Assets/Scripts/InputManager.cs

```diff
@@ -0,0 +1,59 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class InputManager : MonoBehaviour
+{
+    [SerializeField] private Movement _movement;
+    [SerializeField] private MouseLook _mouseLook;
+
+    private PlayerInput _input;
+    private PlayerInput.PlayerActions _playerActions;
+    private PlayerInput.UIActions _uiActions;
+
+    private Vector2 _horizontalInput;
+    private Vector2 _mouseInput;
+
+    public static InputManager instance { get; private set; }
```

Comment:
```suggestion
    public static InputManager s_Instance { get; private set; }
```
[Naming conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names#naming-conventions)